### PR TITLE
extensions.openssl: fix build after https://github.com/nixOS/nixpkgs/commit/6f6090be7711e66091c725527a913ab16e1cd3e0

### PR DESCRIPTION
**Pull request by ajs124**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -21,6 +21,9 @@
         pkgs = import nixpkgs.outPath {
           config = {
             allowUnfree = true;
+            permittedInsecurePackages = [
+              "openssl-1.1.1u"
```
Comment (jtojnar) [inline flake.nix:25]: This may not work when overriding `nixpkgs` input. And when using the overlay. But perhaps the latter is fine.

Comment (ajs124) [reply flake.nix:25]: :thinking: are you sure? Shouldn't this work since this imports nixpkgs via `.outPath`?

Comment (jtojnar) [reply flake.nix:25]: Sorry, I meant that if you override `nixpkgs`, there might be a different version of openssl.

Comment (ajs124) [reply flake.nix:25]: Ah yeah, makes sense, that could happen.

**Comment by drupol** [comment]
I guess we can close this issue now?

**Pull request closed without merging**